wikimedia/wikipedia-ios
# Use number formatter for long character and edit counts in AaaLD experiment

**Phabricator:** https://phabricator.wikimedia.org/T265679

### Notes
* Use our `localizedThousandsStringFromNumber` number formatter logic for characters (added or deleted) and editor edit counts over 1000. 
* The counts in the header of the AaaLD modal are left unchanged.

### Test Steps
Until the survey announcement patch is merged, [this manual patch](https://github.com/wikimedia/wikipedia-ios/pull/3763/commits/d5b599ed32c212c1c1dd5b7f0cecf045e4d66330) is required to see the AaaLD experiment.

1. Observe the in article experiment - confirm that the interstitial character and author edit counts greater than 1000 are formatted as per the `localizedThousandsStringFromNumber`'s K, M, B pattern.
2. Observe the modal - confirm that other than in the page header, the same pattern as step 1 holds.

### Screenshot
<img width="793" alt="number formatter" src="https://user-images.githubusercontent.com/5652327/98619087-5b3cc180-22b7-11eb-87a0-4825cf48c288.png">
